crates/iota-open-rpc/spec/openrpc.json

```diff
@@ -6749,14 +6828,6 @@
           "type"
         ],
         "properties": {
-          "bcs": {
-            "description": "Base 58 encoded bcs bytes of the move event",
```

Comment:
Would it be useful to bring the description back for the 2 variants?

Reply:
Happens because of the flattening, I fear we cannot avoid that without removing that or adjusting annotations.